package/Dockerfile

```diff
@@ -1,6 +1,9 @@
+# syntax=docker/dockerfile:1.7.0
```

Comment:
This syntax is for this error `ERROR: failed to solve: failed to process “${TARGETPLATFORM#linux/}“: unsupported modifier (#) in substitution` when running ./scripts/packge locally.

Reply:
https://docs.docker.com/reference/dockerfile/#environment-replacement